viewer/packages/utilities/src/counterMap.ts

```diff
@@ -16,7 +14,10 @@ export function incrementOrInsertIndex(indexMap: Map<number, number>, index: num
 
 export function decrementOrDeleteIndex(indexMap: Map<number, number>, index: number): void {
   const count = indexMap.get(index);
-  assert(count !== undefined);
+
+  if (count === undefined) {
+    return;
+  }
```

Comment:
Uhm. Can/should this happen?

Reply:
Yes it can, we have some state where the same sector gets discarded twice, or that a sector loads, and then before finish loading gets discarded which then causes this effect.